bricks/next-builder/src/data-providers/chunks/doLintStoryboard.ts

```diff
@@ -355,11 +359,45 @@ export function doLintStoryboard(storyboard: Storyboard): StoryboardError[] {
     }
   );
 
+  if (Array.isArray(routes)) {
+    for (const route of routes) {
+      const injects = new Set<string>();
+      visitStoryboardExpressions(
+        [route],
+        // eslint-disable-next-line
+        (node) => {},
+        {
+          matchExpressionString: (value) => false,
```

Comment:
貌似这样才对？否则可能会错误匹配 ``<% `${abc}` %>``

```suggestion
          matchExpressionString: (value) => true,
```